graphs/kickiip.fbp

```diff
@@ -1,4 +1,5 @@
-# !runtime noflo
+# @runtime noflo-nodejs
+# @name IIPKickTest
```

Comment:
No point whatsoever in regards to newer NoFlo, only there for usage with old NoFlo.